app/IATI/Services/ElementCompleteService.php

```diff
@@ -535,6 +537,10 @@ public function isConditionsElementCompleted($activity): bool
     {
         $this->element = 'conditions';
 
+        if (isset($activity->conditions) && Arr::get($activity->conditions, 'condition_attached') === '0') {
```

Comment:
might be better to define a default value as third parameter so that Arr::get does not throw error

Arr::get($activity->conditions, 'condition_attached', '1') or something